<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewMacro</class>
 <widget class="QMainWindow" name="CreateNewMacro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="MacroSlot1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Unnamed Macro 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MacroSlot2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Unnamed Macro 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MacroSlot3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Unnamed Macro 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MacroLog4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Unnamed Macro 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MacroSlot5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Unnamed Macro 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="UserLogBackground">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>261</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/white background.PNG</pixmap>
    </property>
   </widget>
   <widget class="QScrollBar" name="UserLogScrollBar">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>16</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="BackgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/blue background.PNG</pixmap>
    </property>
   </widget>
   <widget class="QScrollArea" name="UserLogArea">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>261</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>259</width>
       <height>379</height>
      </rect>
     </property>
     <widget class="QLabel" name="UserLog">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>381</height>
       </rect>
      </property>
      <property name="text">
       <string>example</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>BackgroundLabel</zorder>
   <zorder>MacroSlot1</zorder>
   <zorder>MacroSlot3</zorder>
   <zorder>MacroLog4</zorder>
   <zorder>MacroSlot5</zorder>
   <zorder>UserLogBackground</zorder>
   <zorder>MacroSlot2</zorder>
   <zorder>UserLogScrollBar</zorder>
   <zorder>UserLogArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
